### PR TITLE
Model/bug fixing video audio search

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java [hunk 1]
```diff
@@ -30,17 +30,19 @@
     private static List<String> list;
     private static Float searchDuration;
 
+    /**
+     * This method read run exiftool.
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java]: delete "read"

File: src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java [hunk 2]
```diff
@@ -51,17 +53,37 @@ public String readAll() {
...
             }
         } catch (IOException e) {
             e.printStackTrace();
         }
-        return "No available";
+    }
+
+    /**
+     * This method read mime type.
+     * @param mimeType
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java]: explain the param

File: src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java [hunk 3]
```diff
@@ -51,17 +54,37 @@ public String readAll() {
...
         }
-        return "No available";
+    }
+
+    /**
+     * This method read mime type.
+     * @param mimeType string that contains metadata.
+     */
+    public static void mimeType(String mimeType) {
+        String mimetype2 = mimeType;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java]: rename variable mimetype2

File: src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java [hunk 4]
```diff
@@ -51,17 +54,37 @@ public String readAll() {
                 sampleRAte(metadata);
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java]: refactor sampleRAte method

File: src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java [hunk 5]
```diff
@@ -30,38 +30,61 @@
...
         try {
             while ((metadata = stdInput.readLine()) != null) {
                 audioChannel(metadata);
                 audioCodec(metadata);
-                sampleRAte(metadata);
+                sampleRate(metadata);
                 list.add(metadata);
                 duration(metadata);
+                mimeType(metadata);
                 if ((metadata.contains("Read_All"))) {
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/common/MetadataAudioExtractor.java:61]: Empty if, fill it or delete

**Pull request merged**